GetModTime: panic: runtime error: invalid memory address or nil pointer dereference

Seen here: https://storage.googleapis.com/minikube-builds/logs/7075/ce5bf6c/KVM_Linux.html#fail_TestStartStop%2fgroup%2fnewest-cni

```
I0317 14:11:48.704639    4023 main.go:110] libmachine: Making call to close connection to plugin binary

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0xb70e39]
goroutine 1 [running]:
k8s.io/minikube/pkg/minikube/assets.(*FileAsset).GetModTime(0xc000408e10, 0x0, 0x9, 0xc00082c240, 0xc000679201, 0x7b0101)
	/app/pkg/minikube/assets/vm_assets.go:123 +0x59
k8s.io/minikube/pkg/minikube/command.(*SSHRunner).sameFileExists(0xc0000103f8, 0x1d89ac0, 0xc000408e10, 0xc00082c240, 0x24, 0x40, 0x0, 0x0)
	/app/pkg/minikube/command/ssh_runner.go:215 +0x87
k8s.io/minikube/pkg/minikube/command.(*SSHRunner).Copy(0xc0000103f8, 0x1d89ac0, 0xc000408e10, 0x0, 0x0)
	/app/pkg/minikube/command/ssh_runner.go:153 +0x1a8
k8s.io/minikube/pkg/minikube/bootstrapper.SetupCerts(0x1d64b00, 0xc0000103f8, 0xc0002b8260, 0xe, 0x7fffcc000570, 0x2a, 0x1a742f2, 0xa, 0x0, 0x0, ...)
	/app/pkg/minikube/bootstrapper/certs.go:128 +0xf76
k8s.io/minikube/pkg/minikube/bootstrapper/kubeadm.(*Bootstrapper).SetupCerts(0xc000a928a0, 0xc0002b8260, 0xe, 0x7fffcc000570, 0x2a, 0x1a742f2, 0xa, 0x0, 0x0, 0x0, ...)
	/app/pkg/minikube/bootstrapper/kubeadm/kubeadm.go:389 +0xbc
k8s.io/minikube/pkg/minikube/node.setupKubeAdm(0x1d8e5c0, 0xc0002b1e40, 0x7fffcc000570, 0x2a, 0x0, 0xc0004e4300, 0x46, 0x898, 0x2, 0x4e20, ...)
	/app/pkg/minikube/node/config.go:125 +0x4c7
k8s.io/minikube/pkg/minikube/node.Start(0x7fffcc000570, 0x2a, 0x0, 0xc0004e4300, 0x46, 0x898, 0x2, 0x4e20, 0x1a6b53a, 0x4, ...)
	/app/pkg/minikube/node/start.go:86 +0x5e4
k8s.io/minikube/cmd/minikube/cmd.startNode(0xc000781500, 0x7fffcc000570, 0x2a, 0x0, 0xc0004e4300, 0x46, 0x898, 0x2, 0x4e20, 0x1a6b53a, ...)
	/app/cmd/minikube/cmd/start.go:393 +0xe5
k8s.io/minikube/cmd/minikube/cmd.runStart(0x2a45e60, 0xc0004c41c0, 0x1, 0xe)
	/app/cmd/minikube/cmd/start.go:340 +0x628
github.com/spf13/cobra.(*Command).execute(0x2a45e60, 0xc0004c40e0, 0xe, 0xe, 0x2a45e60, 0xc0004c40e0)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830 +0x2aa
github.com/spf13/cobra.(*Command).ExecuteC(0x2a44f60, 0x0, 0x1, 0xc0004e1980)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
k8s.io/minikube/cmd/minikube/cmd.Execute()
	/app/cmd/minikube/cmd/root.go:108 +0x6a4
main.main()
	/app/cmd/minikube/main.go:66 +0xea
** /stderr **
```

--- a/pkg/minikube/assets/vm_assets.go
+++ b/pkg/minikube/assets/vm_assets.go
@@ -120,7 +120,10 @@ func (f *FileAsset) GetLength() (flen int) {
 // GetModTime returns modification time of the file
 func (f *FileAsset) GetModTime() (time.Time, error) {
 	fi, err := os.Stat(f.AssetName)
-	return fi.ModTime(), err
+	if err != nil {
+		return time.Time{}, err
+	}
+	return fi.ModTime(), nil
 }
 
 // Read reads the asset

